source/index.txt

```diff
@@ -73,104 +106,96 @@ suit your needs.
 
    * -
      - Compass
-     - Compass Community
-     - Compass Readonly
      - Compass Isolated
+     - Compass Readonly
+     - Compass Community
 
-   * - View :doc:`documents </documents>`,
-       :doc:`collections </collections>`,
-       :doc:`databases </databases>`, and :ref:`indexes <indexes-tab>`
+   * - Read data from your MongoDB database, including visual query
+       and aggregation builders.
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
 
-   * - Interact with documents, collections, and databases with full
-       CRUD functionality
+   * - Create, update, or delete data from your MongoDB database
      - |checkmark|
      - |checkmark|
-     -
+     - 
      - |checkmark|
 
-   * - Build and run ad hoc queries
+   * - View indexes and related statistics
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
 
-   * - Manage indexes: view stats, create, and delete
+   * - Create or Delete Indexes
      - |checkmark|
      - |checkmark|
-     -
+     - 
      - |checkmark|
 
    * - View existing :ref:`document validation rules <validation>`
      - |checkmark|
-     - 
      - |checkmark|
      - |checkmark|
+     - 
 
-   * - Create, delete and edit document validation rules
+   * - Modify or remove document validation rules
      - |checkmark|
-     - 
-     -
      - |checkmark|
+     - 
+     - 
 
    * - View and optimize query performance with visual explain plans
```

Comment:
Why would Compass Readonly not support this? Seems like Community would be the one to not support this.